### PR TITLE
volume/csi-wrapper: update README doc and fix patch

### LINKED ISSUE #1042: Replay CSI wrapper demo and update documents
To continue the development work for secure storage in csi-wrapper, we need replay following https://github.com/confidential-containers/cloud-api-adaptor/tree/staging/volumes/csi-wrapper based on the latest block storage csi and the cloud-api-adaptor operator.

We also need update new csi-wrapper images urls according to #1030

### PR DESCRIPTION
**Pull request by genjuro214**
- still need update csi driver image tags

fixes #1042

### CODE REVIEW

**Review by liudalibj — APPROVED** [review]
LGTM, thanks @genjuro214

File: volumes/csi-wrapper/README.md [hunk 1]
```diff
@@ -128,8 +128,8 @@ root@csi-x86-worker:~/cloud-api-adaptor#
 
 1. Create the PeerpodVolume CRD object
```
Comment (liudalibj) [inline volumes/csi-wrapper/README.md:129]: L140, should remove `volumes/csi-wrapper/`

**Review by liudalibj — CHANGES_REQUESTED** [review]
L140, should remove volumes/csi-wrapper/

**Review by liudalibj — APPROVED** [review]
LGTM

**Review by huoqifeng — APPROVED** [review]
/lgtm

**Pull request merged**